src/helpers/Cp.php

```diff
@@ -688,39 +730,67 @@ public static function elementCardHtml(ElementInterface $element, array $config
                 Html::endTag('div');
         }
 
-        $thumb = $element->getThumbHtml(128);
-        if ($thumb === null && $element instanceof Iconic) {
-            $icon = $element->getIcon();
-            if ($icon) {
-                $thumb = Html::tag('div', Cp::iconSvg($icon), [
-                    'class' => array_filter([
-                        'cp-icon',
-                        $element instanceof Colorable ? $element->getColor()?->value : null,
-                    ]),
-                    'aria' => ['hidden' => true],
-                ]);
-            }
-        }
+        $thumb = $element->getThumbHtml(120);
+        $icon = $element instanceof Iconic ? $element->getIcon() : null;
+        $title = $element->getCardTitle();
 
         $html = Html::beginTag('div', $attributes) .
+            Html::beginTag('div', ['class' => 'card-titlebar']) .
+            Html::beginTag('div', [
+                'class' => ['flex', 'flex-nowrap', 'flex-gap-xs'],
```

Comment:
```suggestion
                'class' => ['flex', 'flex-nowrap', 'flex-gap-s'],
```
The current spacing seems a little close to me; would this work?

Reply:
Sure, done!